GitTfs/Core/IGitRepository.cs

```diff
@@ -16,7 +16,7 @@ public interface IGitRepository : IGitHelpers
         IGitTfsRemote ReadTfsRemote(string remoteId);
         IGitTfsRemote CreateTfsRemote(RemoteInfo remoteInfo, string autocrlf = null, string ignorecase = null);
         void DeleteTfsRemote(IGitTfsRemote remoteId);
-        IEnumerable<string> GetGitRemoteBranches(string gitRemote);
+        IEnumerable<string> GetGitRepositoryBranchRemotes(string gitRemote);
```

Comment:
Why the method name change? You're changing an interface _contract_ that others may depend upon, potentially breaking 3rd party libraries that build on top of Git-TFS.